.fixtures.yml

```diff
@@ -1,7 +1,9 @@
 ---
 fixtures:
   repositories:
-    apt: https://github.com/puppetlabs/puppetlabs-apt.git
+    apt:
+      repo: https://github.com/puppetlabs/puppetlabs-apt.git
+      tag: v10.0.1
```

Comment:
Vox Pupuli doesn't support Puppet 7 anymore, so we shouldn't be pinning puppetlabs-apt like this. We dropped Puppet 7 support in #745.